core/core-dotspacemacs.el

```diff
@@ -80,6 +80,9 @@ banner, `random' chooses a random text banner in `core/banners'
 directory. A string value must be a path to a .PNG file.
 If the value is nil then no banner is displayed.")
 
+(defvar dotspacemacs-version-check-enable t
```

Comment:
The naming was based off of names in core-release-management.el where there are names like:

```
(defvar spacemacs-version-check-interval "6 hours"
  "Time between two version checks.")
(defvar spacemacs-version-check-lighter "[+]"
  "Text displayed in the mode-line when a new version is available.")
```
https://github.com/tenthousandfailures/spacemacs/blob/disable_auto_update_3438_3/core/core-release-management.el

I think "dotspacemacs-version-check-enable" fits with the code in use today in spacemacs especially in file that the check is going into above.  I think "*version-check-enable" sounds better than "*-version-check-enabled" because "enable" by itself sounds more like an active switch.  "enabled" sounds more like a check on a variable than a switch.